src/main/java/crazypants/enderio/machine/obelisk/render/ObeliskRenderManager.java

```diff
@@ -53,20 +55,20 @@ public void registerRenderers() {
       registerRenderer(block, TileExperienceObelisk.class, eor);
 
     }
-    block = blockAttractor.getBlock();
+    block = blockAttractorObelisk.getBlock();
     if (block != null) {
       ObeliskSpecialRenderer<TileAttractor> eor = new ObeliskSpecialRenderer<TileAttractor>(
-          new ItemStack(itemMaterial.getItem(), 1, Material.ATTRACTOR_CRYSTAL.ordinal()), block);
+          new ItemStack(ModObject.itemMaterial.getItem(), 1, Material.ATTRACTOR_CRYSTAL.ordinal()), block);
```

Comment:
you should never need to use ordinal() for itemstack creation. All item enums have stack getters for each enum: Material.ATTRACTOR_CRYSTAL.getStack()